source/ranges.tex

```diff
@@ -3411,7 +3411,7 @@
     // \ref{range.repeat.iterator}, class \tcode{repeat_view::\exposid{iterator}}
     struct @\exposidnc{iterator}@;                            // \expos
 
-    @\exposidnc{movable-box}@<W> @\exposid{value_}@ = W();                // \expos, see \ref{range.move.wrap}
+    @\exposidnc{movable-box}@<W> @\exposid{value_}@;                      // \expos, see \ref{range.move.wrap}
```

Comment:
This isn't copy-list-initialization, there's no `{}` here - it's plain "copy-initialization". The proposed change isn't editorial, it's normative since it causes the default constructor to default-initialize `value_` instead of value-initializing it. These comments apply to line 13304 as well.

Reply:
Thanks for the correction. I'm going to submit an LWG since it actually [caused some issues](https://godbolt.org/z/14rq9357T).

Reply:
> it causes the default constructor to default-initialize `value_` instead of value-initializing it

I believe default-initialization and value-initialization equivalent for _`movable-box`_. It's default ctor, whenever provided, always value-initializes the contained value.

However, the propose change seems non-editorial for me. The current wording does require move construction in these default ctors (as specified via `optional`'s corresponding ctor), which is probably wrong, and perhaps an LWG issue is needed to correct them.

Reply:
@CaseyCarter , "t causes the default constructor to default-initialize value_ instead of value-initializing it."

This is not quite accurate. We're initializing a `movable-box<W>` with a value-initialized value of type `W`, which may or may not be equivalent to value-initializing the `movable-box<W>` itself (this issue claims it isn't equivalent, and actually harmful).